.travis.yml

```diff
@@ -15,9 +15,10 @@ script:
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run wrangler
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run job-runner
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run patient-care
-  # Then, spin them up normally in daemon mode and confirm they run
-  - docker-compose -f docker-compose.yml up -d
   - docker ps
+  ## Then, spin them up normally in daemon mode and confirm they run
+  #- docker-compose -f docker-compose.yml up -d
```

Comment:
We need that to make sure it all runs. The error was `The job exceeded the maximum time limit for jobs, and has been terminated.` According to Travis docs, this is because `With our current timeouts, a job will be terminated if it's still running after 50 minutes (120 minutes on travis-ci.com)` I think the solution for this is to make the builds take less time. I'll talk to @e-t-k about this.